pkg/smartcontract/manifest/standard/comply.go

```diff
@@ -40,15 +40,12 @@ func Check(m *manifest.Manifest, standards ...string) error {
 func Comply(m, st *manifest.Manifest) error {
 	for _, stm := range st.ABI.Methods {
 		name := stm.Name
-		md := m.ABI.GetMethod(name)
+		md := m.ABI.GetMethod(name, len(stm.Parameters))
 		if md == nil {
 			return fmt.Errorf("%w: '%s'", ErrMethodMissing, name)
```

Comment:
We can also add parameters count to the error message.

Reply:
Fixed.